bin/oref0-setup.sh

```diff
@@ -169,6 +173,19 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
       # Force uppercase, just in case the user entered ww
       radio_locale=${radio_locale^^}
 
+      # check if user has a TI USB stick and a WorldWide pump and want's to reset the USB subsystem during mmtune if the TI USB fails
+      ti_usb_ww0="no" # assume you don't want it by default
```

Comment:
Difference between ti_usb_ww0 and ti_usb_ww1 was the way the python script oref0-subg-ww-radio-parameters.py was called. refactored it to use --ww_ti_usb_reset=[yes|no] for WW-pump which is much cleaner